pkg/api/aim/response/project.go

```diff
@@ -1,5 +1,22 @@
 package response
 
+// ProjectActivity represents the response json for the `GET aim/projects/activity` endpoint.
+type ProjectActivity struct {
+	NumExperiments  float64        `json:"num_experiments"`
+	NumRuns         float64        `json:"num_runs"`
+	NumActiveRuns   float64        `json:"num_active_runs"`
+	NumArchivedRuns float64        `json:"num_archived_runs"`
+	ActivityMap     map[string]int `json:"activity_map"`
+}
+
+// GetProject represents the response json for the `GET aim/projects` endpoint.
+type GetProject struct {
```

Comment:
can you please add `Response` to the end of the name to be consistent with `ProjectParamsResponse` and what we have in `mflow` part?

Reply:
yep